### ISSUE TITLE
[Suggestion] Broken Electric Tool State

### ISSUE THREAD

**Issue by Ethryan** [GregTech6/gregtech6#22]
As discussed on Bears testing stream for 6.14.00.
So you can replace the material tip of the tool without losing everything else.

**Comment by Ethryan** [comment]
Is it really that hard to remove the electric tool tip material data when the material runs out of durability?
And then combine the broken tip version with a new material tip to repair it?

@GregoriusT

**Comment by GregoriusT** [comment]
There is an easy way to do this, but I was a tad bit distracted by Compat stuff and an entire months worth of migraine issues...

**Comment by Ethryan** [comment]
Happy birthday issue #22, hope you get implemented in the year to come.

**Comment by Ethryan** [comment]
Another year has come and gone…

Maybe this can be added soon?,

@GregoriusT

**Comment by Ethryan** [comment]
Another year

**Comment by SilearFlare** [comment]
try pinging greg again

**Comment by GregoriusT** [comment]
why are you trying to make me productive WHILE I'M MELTING IN THE SUMMER HEAT ARRRRRGH

**Comment by Ethryan** [comment]
Stopped melting yet @GregoriusT ?

**Comment by SilearFlare** [comment]
Maybe it'd be wiser to ask again in winter :hand_over_mouth:

**Comment by Ethryan** [comment]
Well... It's cold and Autumn now, he can't possibly be overheating.

**Comment by GregoriusT** [comment]
yeah its finally normal temps now, oh whats that? oh hi @OvermindDL1 , you wanna play Cracktorio with IR3? aaaaaaand that happened again...

**Comment by Ethryan** [comment]
Time for this to be put off again Greg?

**Comment by SilearFlare** [comment]
`issue opened on 2020`
![image](https://github.com/GregTech6/gregtech6/assets/5557920/e4bf883c-54f0-4555-8054-50e779af1515)
I hate to say this but at this point you might be better off making the PR yourself :rofl:

**Comment by Ethryan** [comment]
If I poke you about this again, will it be put off until 2026?
